olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -1316,11 +1316,11 @@ spec:
         - knativeeventings.operator.knative.dev
   relatedImages:
     - name: "knative-operator"
-      image: "registry.ci.openshift.org/knative/release-1.34.0:serverless-openshift-knative-operator"
+      image: "registry.ci.openshift.org/knative/main:serverless-openshift-knative-operator"
```

Comment:
I think this should be release-1.34.1 but not sure why it take main by default @pierDipi @dsimansk is something i missed?

Reply:
There's a env var to set during generator cmd. Just a sec... :)

Reply:
https://github.com/openshift-knative/serverless-operator/blob/main/Makefile#L350-L353

The following should do the trick. 
```
CURRENT_VERSION_IMAGES=release-1.34.1 make generated-files
```

Reply:
^^ @Kaustubh-pande